app/dashboard/templates/profiles/tab_people.html

```diff
@@ -1,27 +1,193 @@
 {% load i18n static avatar_tags humanize %}
 {% if not hidden %}
+<div id="profile-tabs" class="tab-container font-body mt-3 mb-4">
+  <button id="nav-description" href="{{profile.url}}/people?sub=followers" class="text-center section-tab {% if foltab == "followers" %} active {% endif %}">
+    {% trans "FOLLOWERS" %} <span  class="nav-badge">({{all_followers.count}})</span>
+  </button>
+  <button id="nav-description" href="{{profile.url}}/people?sub=following" class="text-center section-tab {% if foltab == "following" %} active {% endif %}">
+    {% trans "FOLLOWING" %} <span  class="nav-badge">({{all_following.count}})</span>
+  </button>
+  {% if profile.is_org %}
+  <button id="nav-description" href="{{profile.url}}/people?sub=core_team" class="text-center section-tab {% if foltab == "core_team" %} active {% endif %}">
+    {% trans "CORE TEAM" %} <span  class="nav-badge">({{team|length}})</span>
+  </button>
+  <button id="nav-description" href="{{profile.url}}/people?sub=leaderboard" class="text-center section-tab {% if foltab == "leaderboard" %} active {% endif %}">
+    {% trans "LEADERBOARD" %}
+  </button>
+  {% endif %}
+</div>
+{% endif %}
 
-  <div class="tab-projects d-flex flex-column mb-4">
-    <h3 class="font-body mb-3">Core Team</h3>
-    {% if team|length == 0 %}
-      <p class="text-center">No Members</p>
-    {% endif %}
-    {% for org in team %}
-      <div class="tab-projects__item d-flex mb-0 {% cycle 'odd' 'even' %} py-2 my-auto">
-        <div class="activity-avatar ml-5">
-          <img height="24" width="24" class="avatar secondary_avatar rounded-circle align-self-end" src="{% avatar_url org %}" />
-        </div>
-        <div class="tab-projects__item-details ml-5 my-auto w-100">
-          <div class="font-body d-inline">
-            <a class="font-body" href="/profile/{{org}}">
-              {{org}}
+<div class="tab-projects d-flex flex-column">
+
+  {% if foltab == 'core_team' %}
```

Comment:
Is this needed?

Reply:
Yes, I think it should be renamed to something common as all the other tabs are using the same css styling too 
![Screenshot 2020-10-12 at 7 41 07 PM](https://user-images.githubusercontent.com/31698676/95756051-e16ae580-0cc2-11eb-9333-b9f745cbd15f.png)